app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,7 @@
+class TweetsController < ApplicationController
+  def profile
+    @tweets = Tweet.order(:created_at)
```

Comment:
Consider moving `order` to a scope in your `Tweet` model. Then you can just do `@tweets = Tweet.newest`